django/core/handlers/asgi.py

```diff
@@ -187,30 +187,40 @@ async def handle(self, scope, receive, send):
             body_file.close()
             await self.send_response(error_response, send)
             return
+
+        async def process_request(request, send):
+            response = await self.run_get_response(request)
+            await self.send_response(response, send)
+
         # Try to catch a disconnect while getting response.
         tasks = [
-            asyncio.create_task(self.run_get_response(request)),
+            # We will check the status of these tasks and (optionally)
```

Comment:
Please omit `We` in comments
```suggestion
            # Check the status of these tasks and (optionally) ...
```